### DISCUSSION TITLE
下拉框值选项为什么不支持划词选中呢？

### DISCUSSION THREAD

**Discussion by yinjialu** [vueComponent/ant-design-vue#6305] [General]
<img width="484" alt="image" src="https://user-images.githubusercontent.com/37073056/220817506-f0682e0d-d992-4515-8ddb-7954e2cd24cf.png">
划词选中是常见的功能，一般是默认支持的，如上图
但是下拉框的选项不支持划词选中拷贝
<img width="481" alt="image" src="https://user-images.githubusercontent.com/37073056/220817669-efb7b97c-8ecb-4707-be27-5a4fd219ddcd.png">

为用户拷贝选项内容带来了麻烦

**Comment by daiwanxing** [comment]
有个可行的办法是为下拉列表中的文本元素添加一个新的 CSS 类，将其 user-select 属性设置为 `auto` 或 `text`

**Reply by yinjialu** [reply]
这个办法应该不行，因为禁止拷贝是通过 js 实现的
![image](https://user-images.githubusercontent.com/37073056/223591098-a008ded1-c49a-48af-b034-b1f6f72bc055.png)
https://vscode.dev/github.com/vueComponent/ant-design-vue/blob/3d2d5fdd996512d9813ec0cc0d7d3cd2e6a0f376/components/vc-select/OptionList.tsx#L55-L58

**Reply by daiwanxing** [reply]
仔细看了下源码后，发现是我想的太简单了，的确给 select 组件的根节点注册了一个 `onMousedown` 事件，这样就除了改源码之外，没有其他办法了。我不知道为啥作者要这样设计的用途是啥🫤。又看了 element-plus 的 select 组件并没有做这样的限制。

**Reply by daiwanxing** [reply]
试了下，目前可行的办法是 安装 `patch-package`, 然后删掉你截图这行代码，最后 `npx patch-package ant-design-vue ` 就行了。
